sdk/iot/core/tests/cmocka/az_iot_sas_token_tests.c

```diff
@@ -249,10 +244,31 @@ void az_iot_sas_token_generate_with_keyname_succeeds(void** state)
   az_span signature = AZ_SPAN_FROM_STR(TEST_SIG);
 
   uint8_t raw_sas_token[256];
-  az_span sas_token
-      = az_span_init(raw_sas_token, 0, _az_COUNTOF(raw_sas_token));
+  az_span sas_token = az_span_init(raw_sas_token, 0, _az_COUNTOF(raw_sas_token));
 
   assert_true(az_succeeded(az_iot_sas_token_generate(
       iothub_fqdn, device_id, signature, expiry_time_secs, key_name, sas_token, &sas_token)));
   assert_true(strncmp(expected_sas_token, (char*)raw_sas_token, az_span_length(sas_token)) == 0);
 }
+
+int test_iot_sas_token()
+{
+  const struct CMUnitTest tests[] = {
+    // SAS Token
+    /*cmocka_unit_test(az_iot_sas_token_get_document_NULL_document_fails),
```

Comment:
Why do we have commented out test code? Should we just remove these?

Reply:
No because we would like to have these tests but must remove them for now due to precondition changes. See #467 for tracking of it.